agents/const.go

```diff
@@ -0,0 +1,3 @@
+package agents
+
+const _agentScratchpad = "agent_scratchpad"
```

Comment:
please comment this

Reply:
Previously gufumt would report an error, but it seems that I have a bad implementation of this, I'll make the changes